swarm/storage/localstore/subscription_pull.go

```diff
@@ -116,11 +117,7 @@ func (db *DB) SubscribePull(ctx context.Context, bin uint8, since, until uint64)
 					}
 				}, &shed.IterateOptions{
 					StartFrom: sinceItem,
-					// sinceItem was sent as the last Address in the previous
-					// iterator call, skip it in this one, but not the item with
-					// the provided since bin id as it should be sent to a channel
-					SkipStartFromItem: !first,
```

Comment:
I think we shouldn't do that. This is breaking many assumptions of how `SubscribePull` works. We should at least discuss this and consider the consequences before changing this :)